src/plans/create/CreatePlanWizardFooter.tsx

```diff
@@ -69,11 +74,16 @@ const CreatePlanWizardFooter: FC<CreatePlanWizardFooterProps> = ({
       <Button
         variant={ButtonVariant.secondary}
         onClick={goToPrevStep}
-        isDisabled={activeStep.id === PlanWizardStepId.General}
+        isDisabled={hasError ?? (activeStep.id === PlanWizardStepId.General || isLoading)}
       >
         {t('Back')}
       </Button>
-      <Button variant={ButtonVariant.primary} onClick={onNextClick}>
+      <Button
+        variant={ButtonVariant.primary}
+        onClick={onNextClick}
+        isDisabled={hasError ?? isLoading}
+        isLoading={isLoading}
```

Comment:
isLoading dont disable the button from click?
i mean if u set isLoading={isLoading}. isDisabled={hasError} isn't enough?